scripts/benchmarks_ci.py

```diff
@@ -40,8 +40,85 @@
 import micro_benchmarks
 
 
+class DotNetPerformanceModes(Action):
+    '''
+    Default: (Currently Tiered)
+
+    NoTiering: (Formerly called full_opt)
+        COMPlus_TieredCompilation=0
+            This includes R2R code, useful for comparison against Default and
+            JitOnly for changes to R2R code or tiering.
+
+    JitOnly: Maybe could be called FullOptJitOnly to make it clear what it does
```

Comment:
NoTiering sounds ok.

For  "JitOnly" probably important to indicate Tiering is also disabled. So perhaps FullyJittedNoTiering

Also is `ZapDisable` no longer required? It even if so, it would still be required pre 3.0.

Reply:
I was thinking this would be useful to compare with tiering for steady-state perf. I believe `ZapDisable` is not needed anymore for CoreCLR, and may not be necessary for other runtimes to get the mode that reaches steady-state perf asap. On .NET Framework would it be useful to specify `ZapDisable` for perf comparisons?

Perhaps there could be a mode called `QuickSteadyState`, which is intended to reach steady-state perf asap, and would just disable tiering and R2R as this mode is currently doing. And if it would be interesting there could also be `JitOnly` that also does `ZapDisable=1`.

Reply:
Should I add `ZapDisable=1` in `FullyJittedNoTiering`? If so, what's the variable? `COMPlus_ZapDisable=1`

Reply:
Yes I think so, and that's right, `COMPlus_ZapDisable=1`.

It doesn't hurt to have it for CoreCLR and if there's a need another mode can be added without the ZapDisable at that time.